src/engine/strat_engine/backstore/blockdevmgr.rs

```diff
@@ -394,6 +394,14 @@ fn initialize(
             };
             match ownership {
                 DevOwnership::Unowned => add_devs.push((dev, (devnode, dev_size, f))),
+                DevOwnership::Multipath(_) => {
```

Comment:
Why `DevOwnership::Multipath` instead of `DevOwnership::Theirs("multipath")`? In both cases the course of action for users of the enum will be the same -- don't allow its use by Stratis.

Reply:
Well now that I've taken out the retrieval of what the actual multipath path actually is, having this as a separate enum value to distinguish that with the corresponding text is not longer useful.  I'll rip this out too.